src/frontend/typer/manuell-journalføring.ts

```diff
@@ -65,3 +65,55 @@ export interface INavnOgIdent {
     navn: string;
     id: string;
 }
+
+export enum JournalpostTittel {
+    SØKNAD_OM_BARNETRYGD_ORDINÆR = 'Søknad om barnetrygd ordinær',
+    SØKNAD_OM_UTVIDET_BARNETRYGD = 'Søknad om utvidet barnetrygd',
+    ETTERSENDELSE_TIL_SØKNAD_OM_BARNETRYGD_ORDINÆR = 'Ettersendelse til søknad om barnetrygd ordinær',
+    ETTERSENDELSE_TIL_SØKNAD_OM_UTVIDET_BARNETRYGD = 'Ettersendelse til søknad om utvidet barnetrygd',
+    TILLEGGSSKJEMA_VED_KRAV_OM_UTBETALING = 'Tilleggsskjema ved krav om utbetaling av barnetrygd og/eller kontantstøtte på grunnlag av regler om eksport etter EØS-avtalen',
+}
+
+export const BrevkodeMap = new Map([
```

Comment:
This should also be a record <JournalpostTittel, string>

Reply:
Ok